yarn-project/simulator/src/avm/opcodes/external_calls.test.ts

```diff
@@ -98,7 +105,7 @@ describe('External Calls', () => {
         successOffset,
         /*temporaryFunctionSelectorOffset=*/ 0,
       );
-      await instruction.execute(context);
+      await instruction.run(context);
```

Comment:
I thought about this a bit: consider `execute` (top level) vs `executeInternal` (protected one).

I know you'd have to change all tests :) does not need to happen (especially not in the same PR). Up to you!

Reply:
That's a rename that VS Code can handle for me, heh. But yeah, probably coming up in another PR!